src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -106,6 +107,16 @@ def get_limit():
             return ()
 
     def ansible_playbook(environment, playbook, *cmd_args):
+        # verifying cchq env ansible version before execution
+        required_ansible_version = environment.public_vars['ansible_version']
+        venv_ansible_version = subprocess.run(["ansible", "--version", "|", "head", "-n1", "|", "awk", "'{print $2}'"], stdout=subprocess.PIPE, text=True)
+        ansible_version = str(venv_ansible_version.stdout).split()[1]
+
+        if (version.parse(ansible_version) < version.parse(str(required_ansible_version))):
+            puts(color_error("Please uninstall current Version "+ansible_version+" and install ansible version "+required_ansible_version+"\n"
+                              "in Cchq env. Else the you ran into issues in fetching credentials from secret manager"))
```

Comment:
How about this as an error message/formatting:

```suggestion
            puts(color_error(f"The version of ansible you have installed ({ansible_version}) is no longer supported."))
            puts(color_notice(f"To upgrade from ansible {ansible_version} to {required_ansible_version} or above you will first have to uninstall the current version (due to an idiosyncratic issue)"))
            puts(color_code("  pip uninstall ansible"))
            puts(color_notice("before re-installing the supported version using your standard method."))
```